modin/experimental/pandas/numpy_wrap.py

```diff
@@ -34,7 +37,7 @@
     import pandas
     import os
 
-    _EXCLUDE_MODULES = [modin, pandas]
+    _EXCLUDE_MODULES.extend([modin, pandas])
```

Comment:
cannot we just do `_EXCLUDE_MODULES = [modin, pandas, real_numpy]`?

Reply:
yes, we can :)